client/jetpack-connect/plans.jsx

```diff
@@ -291,7 +302,23 @@ class Plans extends Component {
 				<PlansGrid
 					{ ...this.props }
 					basePlansPath={ this.props.showFirst ? '/jetpack/connect/authorize' : '/jetpack/connect/plans' }
-					onSelect={ this.props.showFirst || this.props.isLanding ? this.storeSelectedPlan : this.selectPlan } />
+					onSelect={ this.props.showFirst || this.props.isLanding ? this.storeSelectedPlan : this.selectPlan }
+					hideFreePlan={ hideFreePlanTest }
+				>
+					{
+						hideFreePlanTest && (
+							<div className="jetpack-connect__plans-nav-buttons">
```

Comment:
This is the component we use for signup: https://github.com/Automattic/wp-calypso/blob/master/client/signup/step-wrapper/index.jsx#L39.

Not sure if it's compatible or worth replacing?

Reply:
I was also thinking about using that component, but it was adding too much margin, and now with the font size alterations it's creating more trouble than helping.

Reply:
No problem. Sounds good.